sdk/loadtestservice/azure-developer-loadtesting/CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (Unreleased)
```

Comment:
Update the expected release date here.

Reply:
Added 2022-01-15 as December has no release